### ISSUE TITLE
automagic changelogs

### ISSUE THREAD

**Issue by otakup0pe** [otakup0pe/avakas#27]
Similar to how the `bump:foo` stuff works. For each line that matches the regex `^change:\s+(?P=words.+)$` pull `words` into a change log item. When the autobump is made, if the right command line option is specified, prepend to the changelog with

* current version
* the date
* a list of changes

[keep a changelog](https://keepachangelog.com/en/0.3.0/) is a good reference

**Comment by colakong** [comment]
If this is automatic, what will be the source of the list of changes? Git commit messages?

This [tool seems to do that](https://github.com/github-changelog-generator/github-changelog-generator), but relies on people tagging things appropriately for different categories of updates.

**Comment by otakup0pe** [comment]
Yeah was going to use different tags in the commit message as hints. I had seen `github-changelog-generator` but it's github only.  And uses the API. Want to be able to support gitlab and plain ol' git as well.
